ReSharper.FSharp/src/FSharp.Psi/src/IFSharpElementFactory.cs

```diff
@@ -26,6 +26,7 @@ public interface IFSharpElementFactory
 
     IMatchExpr CreateMatchExpr(IFSharpExpression expr);
     IMatchClause CreateMatchClause();
+    IMatchClause CreateMatchClause(string unionCase, bool hasFields);
```

Comment:
Let's try creating a more dummy node here and setting the pattern as a type node?
Something like
```csharp
IMatchClause CreateMatchClause();
```
could produce
```fsharp
| _ -> ()
```
and other features would set everything as they need. I think the most crucial thing to do right here is creating a pattern nodes which could be helpful in "deconstruct" actions or similar things.